src/main/java/com/autohome/controller/DeviceManagementController.java

```diff
@@ -0,0 +1,28 @@
+package com.autohome.controller;
+
+import com.autohome.model.Switch;
+import com.autohome.service.DeviceStateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/room/{roomId}/device/{deviceId}/operate")
+public class DeviceManagementController {
```

Comment:
Change name to DeviceOperationController

Reply:
Done